packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -75,3 +83,31 @@ const memoizedUpdateDisplayName = memoizeFnAll((participantId: string, participa
     return participant;
   }
 });
+
+/**
+ * @private
+ */
+export const isTeamsCallParticipants = (
```

Comment:
In this function we are determining if all of the participants are all teams users? Does a MicrosoftTeamsUserIdentifier match that condition?